app/controllers/view_components_system_test_controller.rb

```diff
@@ -1,8 +1,6 @@
 # frozen_string_literal: true
 
-require "rails/application_controller"
-
-class ViewComponentsSystemTestController < Rails::ApplicationController # :nodoc:
```

Comment:
@edwinthinks I'm curious to know if there was a specific reason for using `Rails:: ApplicationController` in the first place?

Reply:
Hey @Spone ; There is no specific reason for doing it with `ApplicationController`. Based on the issues this PR resolves, this does seem like a better option.

Thanks @ihollander for tackling this!

Reply:
Thanks for your confirmation!